sdk/storage/Azure.Storage.sln

```diff
@@ -0,0 +1,131 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
differs from minver above (14)

FWIW, I've been using VS 2019 (Dev16).  I'm guessing we probably should have no less than VS 2017, as a matter of good NET Standard support, and some features that just don't work well in VS 2015 and earlier.

Reply:
(like .editorconfig support, which was added directly to 2019)

Reply:
The minimum version thing was specific to shared projects which are gone.  Our build agent on Windows uses VS 2017.